request-promise/request-promise.d.ts

```diff
@@ -0,0 +1,29 @@
+// Type definitions for request-promise v0.4.2
+// Project: https://www.npmjs.com/package/request-promise
+// Definitions by: Christopher Glantschnig <https://github.com/cglantschnig/>
+// Definitions: https://github.com/borisyankov/DefinitelyTyped
+
+/// <reference path="../node/node.d.ts" />
+/// <reference path="../form-data/form-data.d.ts" />
+/// <reference path="../request/request.d.ts" />
+/// <reference path="../bluebird/bluebird.d.ts" />
+
+declare module 'request-promise' {
+    import request = require('request');
+    import stream = require('stream');
+    import http = require('http');
+    import FormData = require('form-data');
+
+    export = RequestPromiseAPI;
+
+    function RequestPromiseAPI(options: RequestPromiseAPI.Options): request.Request;
```

Comment:
thanks. I forgot to run `npm test`. Well I put there type `any` since depending if you send `options.simple` true or false you get different values in your returned promise.